frontend/javascripts/viewer/model/reducers/skeletontracing_reducer.ts

```diff
@@ -523,6 +523,17 @@ function SkeletonTracingReducer(state: WebknossosState, action: Action): Webknos
       return setExpandedTreeGroups(state, (group: TreeGroup) => expandedGroups.has(group.groupId));
     }
 
+    case "EXPAND_CORRESPONDING_TREE_GROUP": {
+      const { tree } = action;
+      if (tree.groupId == null) {
+        return state;
+      }
+      return setExpandedTreeGroups(
+        state,
+        (group: TreeGroup) => group.isExpanded || group.groupId === tree.groupId,
```

Comment:
this seems to only expand the group which has the tree as a direct child. however, if a tree is nested within several groups, all parent groups have to be expanded. I would have expected that `additionallyExpandGroup` is used here (similar to the other code place).

for that reason, I'd also rename the action to `expandParentGroupsOfTree`. that action could then also be dispatched in tree_hierarchy_view.tsx (basically moving the code from there to this reducer).

Reply:
>this seems to only expand the group which has the tree as a direct child.

have you tested this? for me this actually expands all parent groups currently (I tested this scenario during developing). but it does create an inconsistent state where the store state !== the UI state, so obviously you're right. I didnt find the reason for this behaviour right away but I suppose it doesnt really matter so I'll just fix it

Reply:
> have you tested this?

no, I haven't tested it. my comment is only because of my understanding of the code.

> I'll just fix it

thanks :)